lib/cdk/utils/imageLambda.ts

```diff
@@ -8,9 +8,14 @@ export interface SetupImageLambdaProps {
 	assetsBucket: Bucket
 	layerPath: string
 	lambdaHash: string
+	memory?: number
```

Comment:
I'm thinking if we should go with optional parameters or enforce them.
Personally my preference would be to enforce, especially considering the logic in parent that uses `proces.env` OR default. Open to discussion

Reply:
I would vote for enforcing if the only entrypoint were the CLI. With the introduction of the `CdkUtils` class, I think the "batteries included but replaceable" approach is better.